src/index.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html>
+<head>
+    <title>Idea Town</title>
+    <!-- inject:css -->
+    <!-- endinject -->
+</head>
+<body>
+  <!-- container div -->
+  <div id="container">
+    <header>
+      <img id="logo">
+      <div id="avatar-or-signin-button"></div>
+    </header>
+    <div id="hero-space">
+      <!-- not idea town user? show idea town sales pitch.
+           idea town user? show something exciting and fun -->
```

Comment:
I appreciate these comments, they are very helpful for visualizing the content and providing context.